tests/test_relations.py

```diff
@@ -98,8 +99,23 @@ def test_from_name(self, _find_impl, join, relation_to_role_and_interface):
         self.assertEqual(res.conversations(), ['conv.join'])
 
     @mock.patch.dict('sys.modules')
+    @mock.patch.object(relations.Conversation, 'join')
+    @mock.patch.object(relations, 'hookenv')
+    def test_cold_import(self, hookenv, conv_join):
+        tests_dir = os.path.dirname(__file__)
+        hookenv.charm_dir.return_value = os.path.join(tests_dir, 'data')
+        sys.modules.pop('relations.hyphen-ated.peer', None)
```

Comment:
I started with `del`, but that raises an exception if the value isn't in there, while this doesn't (due to the default `None`).